packages/postgres/src/sql-template.ts

```diff
@@ -1,31 +1,31 @@
+import { type CompiledQuery, init, PostgresDialect } from '@sejohnson/tql';
 import { VercelPostgresError } from './error';
 
-export type Primitive = string | number | boolean | undefined | null;
+const { query, ...rest } = init({ dialect: PostgresDialect });
```

Comment:
From the docs of `@sejohnson/tql` I think the helper functions coming out of `init({ dialect: PostgresDialect })` are all upper-case but they are being exported in `packages/postgres/src/index.ts` as lower-case